vnc.html

```diff
@@ -206,6 +206,9 @@ <h1 class="noVNC_logo" translate="no"><span>no</span><br />VNC</h1>
                     <li>
                         <label><input id="noVNC_setting_view_only" type="checkbox" /> View Only</label>
                     </li>
+                    <li>
+                        <label><input id="noVNC_setting_show_dot" type="checkbox" /> Pointer Always Visible</label>
```

Comment:
Bike shedding, but I wonder if people might think this will show a normal arrow pointer. For reference RealVNC (and derived software) displays "Show dot when no cursor".

Reply:
agreed on that point